tests/test_opusfilter.py

```diff
@@ -1126,3 +1126,107 @@ def test_expand_parameters_invalid(self):
         for case in [Var('unk'), VarStr('{}'), VarStr('{unk}'), VarStr('{mystr}-{unk}')]:
             with self.assertRaises(ConfigurationError):
                 self.of._expand_parameters(case, variables)
+
+
+@unittest.skipIf('varikn' not in globals() or os.environ.get('EFLOMAL_PATH') is None, 'varikn or eflomal not found')
+class TestParallel(unittest.TestCase):
```

Comment:
This is currently failing for me, but seems that the problem is also in the master branch, so I'll look into it myself.

However, it would be good to have some (simpler) test without the need of varikn and eflomal models, so that the CI pipelines would run it. As `test_opusfilter.py` starts to be quite big, you could move this to `test_parallel.py` and add at least one such simpler test there.

Reply:
That failure was simply due to extra argument for the word alignment. I don't know how I'm missed that before...

Please rebase on develop, and remove `'tokenizer': 'none'` from `train_alignment` and `WordAlignFilter` methods also in your tests.

Reply:
Yep, I found this issue. The configuration in test case is outdated.  I will rebase it.